### ISSUE TITLE
Restore Git history

### ISSUE THREAD

**Issue by xurizaemon** [thinktandem/migrate_booster#1]
Hi there

This project's history starts at 2c0a003df2f54f9fa8b7318813705a356d37227a, which means that the history of the [project as shown on Drupal.org](https://git.drupalcode.org/sandbox/onkeltem-2828817) isn't reflected - including the original authorship.

If you clone the project from Drupal.org and then check out your changed content over the top and commit it, you can show a more accurate representation of the history - and your project will be more readily able to contribute improvements back to the original.

Hit me up if you need assistance with this, happy to help!

Looks like a great project to support, and thanks for the blog post mentioning it.

**Comment by labboy0276** [comment]
@xurizaemon correct, I just downloaded the module and then added it here with my changes.

If you want to put in a PR to add the history back, that would be great.
